src/styles.css/button.css

```diff
@@ -0,0 +1,25 @@
+.card__button {
+  display: flex;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
+  background: rgb(0, 172, 220);
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  color: rgb(255, 255, 255);
+  align-self: center;
+  align-items: center;
+  cursor: pointer;
+  text-decoration: none;
+  background-color: var(--acent-color);
+}
+
+.card__button:hover {
+  border: 1px solid rgba(0, 172, 220, 1);
```

Comment:
[STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover